x/concentrated-liquidity/incentives.go

```diff
@@ -118,7 +118,7 @@ func (k Keeper) getInitialUptimeGrowthOutsidesForTick(ctx sdk.Context, poolId ui
 // Returns error if a canonical pool ID exists but there is an issue when retrieving the pool assets for this pool.
 //
 // CONTRACT: canonical Balancer pool has the same denoms as the CL pool and is an even-weighted 2-asset pool.
-func (k Keeper) prepareBalancerPoolAsFullRange(ctx sdk.Context, clPoolId uint64) (uint64, sdk.Dec, error) {
+func (k Keeper) getCanonicalBalancerPoolIDWithFullRangeShares(ctx sdk.Context, clPoolId uint64) (uint64, sdk.Dec, error) {
```

Comment:
This is mutative so let's keep this as non-getter name please. Also, godoc would have to be updated

Reply:
Good call that it's mutative thus should remain as is, reverted back to original method name although I do think that "prepare" is quite ambiguous, but do not have better options for this